### DISCUSSION TITLE
EK Nucleus

### DISCUSSION THREAD

**Discussion by SidOfTheNorth** [Rem0o/FanControl.Releases#1740] [Q&A]
I have ordered a EK Nucleus, will it work with FanControl? Both pump, fans and temp sensor?

**Comment by Fenix66d** [comment]
I have the EK Nucleus CR360 Lux installed. If AIO Fans are directly connected to CPU Fan header on the MB, FanControl sees these fans and you can adjust/save your fan curve. FanControl sees the Pump RPM speed, but I have been unable to access this FAN/Pump data to rename the Fan#2 to AIO Pump and set adjusted temp curve. I'm getting an extensive number of speed and sensor errors in my attempts to setup FanControl. Additionally, FanControl doesn't even see my Gigabyte RX 6900 XT video card.
